web/src/pages/order/index.page.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import TextInput from "@/components/organisms/TextInput";
+import { useHooks } from "../login/hooks";
+import StatusSelect from "@/components/organisms/StatusSelect";
+import { Button } from "flowbite-react";
+import CommentInput from "@/components/organisms/CommentInput";
+import ImgUpload from '@/components/organisms/ImgUpload';
+
+const Order = () => {
+    const { formState, handleSubmit } = useHooks();
+
+    return (
+        <section className="bg-white dark:bg-gray-900">
+            <div className="md:h-screen lg:py-0 flex flex-col">
+                <h1 className="text-xl font-bold text-gray-800 text-center">
+                    PC BEE ORDERING PARTS SYSTEM
+                </h1>
+                <div className="grid grid-cols-1 px-4 pt-6 xl:grid-cols-3 xl:gap-4 dark:bg-gray-900">
+                    <div className="col-span-2">
+                        <div className="p-4 mb-4 bg-white border border-gray-200 rounded-lg shadow-sm 2xl:col-span-2 dark:border-gray-700 sm:p-6 dark:bg-gray-800">
+                            <form onSubmit={handleSubmit}>
+                                <div className="grid grid-cols-6 gap-6">
+                                    <div className="col-span-6 sm:col-span-3">
+                                        <TextInput label='Date' name='created_at' {...formState} />
```

Comment:
1. remove the form state for now since we dont have use-form for this page yet. unless you will create a new hook for this page

2. Date should use Date picker, not `TextInput `
```suggestion
                                        <TextInput label='Date' name='created_at' />
```